fxa-auth-db-server/docs/DB_API.md

```diff
@@ -249,7 +250,7 @@ Parameters.
 
 Each token takes the following fields for it's create method respectively:
 
-* sessionToken : data, uid, createdAt
+* sessionToken : data, uid, createdAt, userAgent, os, deviceType
```

Comment:
Is `userAgent` here the full header or something parsed from it?  If the later, let's call it something other than "userAgent" for clarity.  Perhaps "application" or "browser"? 

Is it worth naming these in a way that indicates where this information came from, e.g. "uaApplication", "uaOS", "uaDeviceType"?  Or perhaps we should be factoring them out into a separate table somehow.

Naming is hard.  @philbooth let's try to chat about this early next week before I break for PTO.